lib/ansible/playbook/task.py

```diff
@@ -259,20 +259,10 @@ def preprocess_data(self, ds):
             elif k.startswith('with_') and k.replace("with_", "") in lookup_loader:
                 # transform into loop property
                 self._preprocess_with_loop(ds, new_ds, k, v)
+            elif C.INVALID_TASK_ATTRIBUTE_FAILED or k in self._valid_attrs:
+                new_ds[k] = v
             else:
-                # pre-2.0 syntax allowed variables for include statements at the top level of the task,
-                # so we move those into the 'vars' dictionary here, and show a deprecation message
-                # as we will remove this at some point in the future.
-                if action in C._ACTION_INCLUDE and k not in self._valid_attrs and k not in self.DEPRECATED_ATTRIBUTES:
```

Comment:
I can't find use of `DEPRECATED_ATTRIBUTES` beyond this now deleted reference. Can we just remove it from the base class in `playbook/base.py` now?

Reply:
we can, but i would do in diff PR